<launch>
	<node pkg="performance_tests" type="cpp_talker" name="cpp_talker">
		<param name="hertz" type="int" value="18000"/>
	</node>

	<node pkg="performance_tests" type="cpp_listener" name="cpp_listener"/>
</launch>
